coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/budget/core/ProposalBudgetForm.java

```diff
@@ -202,4 +203,12 @@ public void setDataValidationItems(List<DataValidationItem> dataValidationItems)
 		this.dataValidationItems = dataValidationItems;
 	}
 
+	public boolean isViewOnly() {
+		return viewOnly;
+	}
+
+	public void setViewOnly(boolean viewOnly) {
+		this.viewOnly = viewOnly;
+	}
```

Comment:
Do we really need an additional form property for this? Can't we do this using the view.readOnly or just form.canEditView?